airflow/executors/base_executor.py

```diff
@@ -22,6 +22,7 @@
 
 from airflow import LoggingMixin, conf
 from airflow.models import TaskInstance
+from airflow.models.queue_task_run import QueueTaskRun
```

Comment:
Something about this name doesn't quite sit right with me. How about "QueuedTask", "QueuedExecution", as a name instead? I'm not happy with those either really, but something about "Run" made me think this was more than a class containing the command/task info to run in the queue.

Also not sure it belongs in "airflow.models" -- I guess it sort of is a model, it's just that most other classes in there are DB backed. What do you think about putting this class somewhere under airflow.executors?

Reply:
I will introduce a different name in the final PR. What do you think about RawTaskDeferredRun, LocalTaskJobDeferredRun?